policyengine_us/parameters/gov/states/mi/tax/income/credits/eitc/rate/match.yaml

```
@@ -0,0 +1,16 @@
+description: Michigan matches the federal eitc at the following rate.
+
+metadata:
+  unit: /1
+  label: Michigan EITC match 
+  reference:
+    - title: INCOME TAX ACT OF 1967 - Section 206.272, (1)
+      href: http://www.legislature.mi.gov/(S(ttmppzfvkzs3qs04dtrpwcfp))/mileg.aspx?page=getObject&objectName=mcl-206-272-amended&highlight=income%20AND%20tax
+    - title: 2022 MICHIGAN Individual Income Tax Return MI-1040 Form Page 2 Line 27
+      href: https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/MI-1040.pdf?rev=94571ffae1b04033ad368668974dec7a&hash=837A1A7BC58EE291E23535581A7F88DC    
+values:
+  2008-01-01: 0.1
+  2009-01-01: 0.2
+  2012-01-01: 0.06
+  2023-01-01: 0.3
```

Then move this up to `eitc/match.yaml` and remove the `additional.yaml` file.
```suggestion
  # Reflected starting with 2023 tax form.
  # In 2022, Michigan enacted a supplemental 24% match after tax filing season. 
  2022-01-01: 0.3
```